Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6-slim
+
+WORKDIR /bot
+
+COPY requirements.txt \
+        setup.py \
+        LICENSE \
+        README.md \
+        /bot/
+COPY dorpsgek_github /bot/dorpsgek_github
+
+RUN pip install -r requirements.txt
+RUN pip install -e .
+
+# Validate that what was installed was what was expected
+RUN pip freeze 2>/dev/null | grep -v 'dorpsgek-github' > requirements.installed \
+        && test "`diff -u --strip-trailing-cr requirements.txt requirements.installed | wc -l`" = "0" \
+        || ( \
+            echo "!! ERROR !! different packages installed than defined in requirements.txt:" \
+            && diff -u --strip-trailing-cr requirements.txt requirements.installed \
+            && exit 1 ) 1>&2
```

Comment:
Overly complicated, `diff -uw foo bar` is all that's required - no need for test

Could also combine the diffs - `diff -wu requirements.txt requirements.installed || (echo "Error..." && exit 1)`

Reply:
Though I wonder why the test is even necessary - don't feel like we're expected to be checking pip for bugs

Reply:
We are not checking pip for bugs. We are checking if `requirements.txt` is installed as suggested. Not that someone fucked up `requirements.txt`. So we are checking the developer :)